nautobot/ui/src/components/Apps.js

```diff
@@ -63,6 +53,20 @@ function get_components() {
                     }
                 );
             }
+            // Add Plugin Provided View Components
+            if (value?.default?.routes) {
+                // eslint-disable-next-line
+                value?.default?.routes.forEach(({ groups }) => {
+                    groups.forEach(({ items }) => {
+                        items.forEach(({ namespace, component }) => {
+                            const route = `${slugify(app_name)}:${namespace}`;
+                            base["OverrideViews"][route] = lazy(() =>
```

Comment:
Why `OverrideViews`? This PR doesn't (AIUI) have anything to do with overriding base views, it's about adding entirely new views right?